### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by zhaijianyang** [open-mmlab/mmaction2#751]
I want to retrain the I3D model and modify the loss function. Different epochs use different loss functions. Where should I modify the loss function?

**Comment by congee524** [comment]
you can modify the config of `I3DHead`

```python
model = dict(
    ...
    cls_head=dict(
        type='I3DHead',
        num_classes=400,
        in_channels=2048,
        # modify the loss_cls
        loss_cls=dict(type='CrossEntropyLoss'),
        spatial_type='avg',
        dropout_ratio=0.5,
        init_std=0.01),
    ...)
```

**Comment by dreamerlin** [comment]
If you want to use different loss functions in different epochs, you may need to customize a Hook and rewrite the [`before_train_epoch()`](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/hook.py#L27-L28) function. You can get the epoch number by `runner.epoch` and change the loss function in the `runner.model`

**Comment by congee524** [comment]
Well, I didn't notice that you wanna use different loss functions in different epochs...

Pls, refer to @dreamerlin 's suggestion.
